### PR TITLE
added pressure drop equations in radiator model

### LINKED ISSUE #586: Pressure drop for RadiatorEN442_2
Currently `Annex60.Fluid.HeatExchangers.Radiators.RadiatorEN442_2` does not allow to specify a pressure drop. Shouldn't this be added?

### PR DESCRIPTION
**Pull request by Mathadon**
this closes #586 

unit tests in Anenx60.Fluid were ok

### CODE REVIEW

File: Annex60/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo [hunk 1]
```diff
@@ -41,6 +41,35 @@ model RadiatorEN442_2 "Dynamic radiator for space heating"
   parameter Boolean homotopyInitialization = true "= true, use homotopy method"
     annotation(Evaluate=true, Dialog(tab="Advanced"));
 
+  parameter Boolean use_dh = false
```
Comment (mwetter) [inline Annex60/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo]: For radiators, it does not make sense to use the hydraulic diameter. Therefore, this parameter should be removed.

File: Annex60/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo [hunk 2]
```diff
@@ -41,6 +41,35 @@ model RadiatorEN442_2 "Dynamic radiator for space heating"
...
+  "= true, use dh and ReC, otherwise use deltaM"
+       annotation(Evaluate=true,
+                  Dialog(group = "Transition to laminar",
+                         enable = not linearized));
+  parameter Modelica.SIunits.Length dh=1 "Hydraulic diameter"
```
Comment (mwetter) [inline Annex60/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo]: As above, remove `dh`

File: Annex60/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo [hunk 3]
```diff
@@ -190,20 +234,23 @@ initial equation
    end for;
 
 
+
```
Comment (mwetter) [inline Annex60/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo]: remove empty lines

**Comment by Mathadon** [comment]
I think I addressed the suggestions! I reran unit tests, which were fine.

**Pull request merged**